docs/workflows/remote-profiling.md

```diff
@@ -0,0 +1,123 @@
+<%(TOC)%>
+
+# Remote profiling
+
+<%(Callout message="
+This page discusses how to connect the [Unity Profiler](https://docs.unity3d.com/Manual/Profiler.html) to a worker running in a cloud deployment. 
+
+Before reading this page, make sure you are familiar with:
+
+  * [Build configuration]({{urlRoot}}/modules/build-system/build-config)
+  * [Deploying]({{urlRoot}}/modules/deployment-launcher/overview)
+")%>
+
+The [Unity Profiler](https://docs.unity3d.com/Manual/Profiler.html) offers the ability to connect to an instance of your game running on a different machine. We can utilize this to profile a Unity worker running in a cloud deployment.
+
+## 1. Configure your workers
+
+The Unity Profiler connects to your worker via a host and port combination. The worker will only expose this port if it is built as a `Development Build`. We can configure this via the [build configuration]({{urlRoot}}/modules/build-system/build-config) asset.
+
+1. Find and open the instance of the [build configuration]({{urlRoot}}/modules/build-system/build-config) asset in the Unity Editor Inspector.
+2. Expand the section for the worker type that you want to profile.
+3. Expand the `Cloud Build Options` section within that worker type.
+4. Click on the `Linux` tab to configure the `Linux` build options.
+5. Make sure that the `Development` box is ticked.
+
+<%(#Expandable title="What should it look like when you're done?")%>
+
+![]({{assetRoot}}assets/workflows/remote-profiling-build-config.png)
+
+<%(/Expandable)%>
+
+## 2. Build the workers
+
+Now that the worker is configured correctly to allow remote profiling, lets build our workers.
+
+1. Select **SpatialOS** > **Build for cloud** > **All workers** from the Unity Editor menu to build all workers.
+
+> Alternatively, if you don't need to rebuild all the workers, you can only build the worker type for which you enabled the `Development` option.
+
+## 3. Upload worker assemblies & launch deployment
+
+Use the Deployment Launcher feature module to upload the built out worker assemblies and launch a deployment.
+
+See the [feature module documentation]({{urlRoot}}/modules/deployment-launcher/overview) if you are unsure on how to do this.
+
+## 4. Find the profiling port
+
+At this point, your deployment should be up and running and your workers connected to that deployment.
+
+Since your workers are running in the cloud, we will need to forward the port that is exposed by those workers to our local machine in order to connect the profiler to it. Each worker should log a single message to SpatialOS detailing which port it is currently exposing.
+
+1. Open the deployment in the SpatialOS console.
+2. Navigate to the logs for your deployment.
+3. Look for a message similar to the following: `[UnityGameLogic0:UnityGameLogic0] Unity PlayerConnection port: 55203.`
```

Comment:
one more suggestion :)
Is there an easy way to filter for these kind of messages? as depending on the game the log page might contain a lot of logs already

Reply:
You can search for `PlayerConnection`? Do we want to mention that?